independent-publisher-connectors/SOS Inventory/apiProperties.json

```diff
@@ -0,0 +1,38 @@
+{
+	"properties": {
+		"connectionParameters": {
+			"token": {
+				"type": "oauthSetting",
+				"oAuthSettings": {
+					"identityProvider": "oauth2",
+					"clientId": "[[DUMMY]]",
+					"ClientSecret": "[[DUMMY]]",
+					"scopes": [],
+					"redirectUrl": "https://logic-apis-[parameters('location')].consent.azure-apim.net/redirect",
+					"properties": {},
+					"customParameters": {
+						"authorizationUrl": {
+							"value": "https://api.sosinventory.com/oauth2/authorize"
+						},
+						"tokenUrl": {
+							"value": "https://api.sosinventory.com/oauth2/token"
+						},
+						"refreshUrl": {
+							"value": "https://api.sosinventory.com/oauth2/token"
+						}
+					}
+				}
+			}
+		},
+		"capabilities": [],
+		"description": "Retrieves and updates information in SOS Inventory",
+		"displayName": "SOS Inventory",
```

Comment:
@Harold-Anderson please remove these properties, they are not required in [the template](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/Independent%20Publisher/apiProperties.json).

Reply:
I've made those changes and pushed them to my fork.  What do I do now to get them into this PR?

Reply:
Thank you very much! Your PR was automatically updated when you pushed your changes. We will review updated files and get back with feedback.